bucketlist/models.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime
+from flask import url_for
+from werkzeug.security import generate_password_hash, check_password_hash
+from bucketlist import db
+from bucketlist.exceptions import ValidationError
```

Comment:
I'll leave this one as is to be inline with pep8:
1)standard library imports
2)related third party imports
3)local application/library specific imports